types/tizen-common-web/tslint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules":{
+        "no-const-enum": false,
+        "max-line-length": false,
+        "strict-export-declare-modifiers": false,
```

Comment:
It'd be nice if this could be cleaned up and this exception removed before merge. `declare interface` seems particularly egregious to me - there's no reason for an `interface` to have a `declare` modifier, ever.

Reply:
I understood. Please check the latest commit.